### PR TITLE
Version 1.1.0 release

### PR DESCRIPTION
**Pull request by cmsquared**

### CODE REVIEW

**Comment by certik** [comment]
The tests failed with:
```
Sending build context to Docker daemon 48.62 MB
Error response from daemon: Unknown instruction: FI
```

**Comment by certik** [comment]
It probably fails on this line: https://github.com/laristra/portage/pull/6/files#diff-ebacf6f6ae4ee68078bb16454b23247dR44, probably the Dockerfile has a syntax error.

**Comment by certik** [comment]
The diff is too big, but here is the relevant part of `docker/Dockefile` that is probably a syntax error:
```diff
+RUN if [ ${DOCS} ]; then \
+  make doxygen;
+fi 
```

**Review by cferenba — APPROVED** [review]

**Review by junghans — APPROVED** [review]
This is a bit much too review severely, please break it up in some smaller pull requests next time ;-)

File: .travis.yml [hunk 1]
```diff
@@ -23,10 +27,21 @@ script:
...
                  --build-arg TRAVIS_COMMIT=${TRAVIS_COMMIT} --build-arg COVERAGE=${COVERAGE}
+                 --build-arg DOCS=${DOCS}
                  -t ${TRAVIS_REPO_SLUG}:latest ${HOME}/docker/
 
 after_success:
   - if [[ ${DOCKERHUB} = true && ${DOCKER_USERNAME} && ${DOCKER_PASSWORD} && ${TRAVIS_PULL_REQUEST} == false && ${TRAVIS_BRANCH} == master ]]; then
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push "${TRAVIS_REPO_SLUG}:latest";
     fi
+  - if [[ ${DOCS} = true ]]; then
```
Comment (junghans) [inline .travis.yml]: `[[ ${DOCS} ]]` would work, too.

File: .travis.yml [hunk 2]
```diff
@@ -23,10 +27,21 @@ script:
                  --build-arg TRAVIS_PULL_REQUEST=${TRAVIS_PULL_REQUEST} --build-arg TRAVIS_JOB_ID=${TRAVIS_JOB_ID}
                  --build-arg TRAVIS_TAG=${TRAVIS_TAG} --build-arg TRAVIS_REPO_SLUG=${TRAVIS_REPO_SLUG}
                  --build-arg TRAVIS_COMMIT=${TRAVIS_COMMIT} --build-arg COVERAGE=${COVERAGE}
+                 --build-arg DOCS=${DOCS}
                  -t ${TRAVIS_REPO_SLUG}:latest ${HOME}/docker/
 
 after_success:
   - if [[ ${DOCKERHUB} = true && ${DOCKER_USERNAME} && ${DOCKER_PASSWORD} && ${TRAVIS_PULL_REQUEST} == false && ${TRAVIS_BRANCH} == master ]]; then
```
Comment (junghans) [inline .travis.yml:34]: You only compile with gcc, right? (If you ever enable `clang` you need to check it here.)

File: README.md [hunk 3]
```diff
@@ -1,101 +1,247 @@
...
+
+```c++
+# machine=ml-fey
+module load intel/17.0.1 openmpi/1.10.5 cmake
+JALI_INSTALL_PREFIX=/usr/projects/ngc/private/jali/0.9.8-intel-17.0.1-openmpi-1.10.5
+mkdir build
+cd build
+cmake \
+    -D CMAKE_C_COMPILER=`which mpicc` \
+    -D CMAKE_CXX_COMPILER=`which mpiCC` \
```
Comment (junghans) [inline README.md:171]: The `which mpicc` isn't needed, see 95327cb9e559ffeb1bd55c1d5629bbb5a19c1d66.

File: .travis.yml (hunk 2 above)
Comment (cmsquared) [reply .travis.yml:34]: That's correct, but good point.

File: README.md (hunk 3 above)
Comment (cmsquared) [reply README.md:171]: Ah, good to know.

**Comment by cmsquared** [comment]
fixing build-env now

**Comment by cmsquared** [comment]
actually, @junghans I don't seem to have permissions to create a branch on portage-buildenv in order to issue a PR.  Can you update the Dockerfile there to install `liblapack-dev` and `liblapacke-dev` ? or give me permissions

**Comment by junghans** [comment]
@cmsquared: you got it (the permissions).

**Comment by cmsquared** [comment]
thanks -- getting this to work now, if I ever stop fat-fingering things...

**Comment by certik** [comment]
Congrats @cmsquared. Sorry that you missed all the :beers:.

**Pull request merged**